pkgs/development/ocaml-modules/uucp/default.nix

```diff
@@ -1,30 +1,45 @@
-{ lib, stdenv, fetchurl, ocaml, findlib, ocamlbuild, topkg, uchar, uutf, uunf }:
+{ lib, stdenv, fetchurl, ocaml, findlib, ocamlbuild, topkg, uchar, uutf, uunf, uucd }:
 
 let
   pname = "uucp";
-  version = "11.0.0";
+  version = "13.0.0";
   webpage = "https://erratique.ch/software/${pname}";
+  minimumOCamlVersion = "4.03";
+  doCheck = true;
 in
 
-assert lib.versionAtLeast ocaml.version "4.01";
+assert lib.assertMsg (lib.versionAtLeast ocaml.version minimumOCamlVersion)
+  "${pname} needs at least OCaml ${minimumOCamlVersion}";
```

Comment:
It seems that now `nix-env -qaP` unconditionally outputs:

> trace: uucp needs at least OCaml 4.03

Reply:
Hm, I don't get it with `nix-env -qaP -f ~/src/nix/nixpkgs` and nix 2.3.10.

Reply:
What about `nix-env -qaP -f ~/src/nix/nixpkgs -A ocaml-ng.ocamlPackages_4_02 > /dev/null`?

Reply:
Completely silent for me and lists uucp.

Reply:
> Completely silent for me and lists uucp.

This is the behavior before this PR. After, the `uucp` attribute should be gone.

> There's no reason why qaP would evaluate uucp right? it doesn't need meta or similar afaik…

In order to print the names of the derivations, `nix-env` must evaluate their `name` attribute, hence go through the failing assert.

Reply:
Wait, I'm stupid, I was on the wrong branch. Probably not good UX. I wonder why trace and throw messages are treated differently by `nix-env`, but I'll make another PR reverting this change anyways.